benchmark/src/main/scala/keycloak/scenario/admin/CreateDeleteRealms.scala

```diff
@@ -0,0 +1,15 @@
+package keycloak.scenario.admin
+
+import keycloak.scenario.{CommonSimulation, KeycloakScenarioBuilder}
+
+
+class CreateDeleteRealms extends CommonSimulation {
+
+  setUp("CRUD realms", new KeycloakScenarioBuilder()
```

Comment:
rename this to "Create, Delete Realms" or add the rest of the Update and Read operations in the simulation to List and/or Update the Realms

Reply:
Label changed.

Reply:
perfect